ambari-server/src/main/python/ambari_server/setupSecurity.py

```diff
@@ -79,6 +79,7 @@
 PAM_CONFIG_FILE = 'pam.configuration'
 
 IS_LDAP_CONFIGURED = "ambari.ldap.authentication.enabled"
+IS_LDAP_CONFIGURED_IN_AMBARI_PROPERTIES = "ambari.ldap.isConfigured"
```

Comment:
Why do we need to check the ambari.properties file.  The values stored in the Ambari DB should be good enough.

Reply:
Did not you tell, that it should be backward compatible?

Reply:
Fixed

Reply:
It should be backwards compatible from the perspective of the user.  However the relevant data can all be stored in the Ambari DB so it is in one place.   By retrieving the properties from the Ambari DB rather that looking at the ambari.properties file should suffice.

Reply:
Fixed